instrumentation/lettuce-5.0/src/main/java/com/nr/fit/lettuce/instrumentation/NRErrorConsumer.java

```diff
@@ -0,0 +1,24 @@
+package com.nr.fit.lettuce.instrumentation;
+
+import java.util.function.Consumer;
+
+import com.newrelic.api.agent.NewRelic;
+
+public class NRErrorConsumer implements Consumer<Throwable> {
+
+	private NRHolder holder = null;
+
+	public NRErrorConsumer(NRHolder h) {
+		holder = h;
+	}
+	
+	@Override
+	public void accept(Throwable t) {
+		NewRelic.noticeError(t);
+		if(holder != null && !holder.hasEnded()) {
```

Comment:
Doesn't look formatted.

Reply:
yeah, I think I need to format 4.3 and 5 source